lib/protocol.js

```diff
@@ -199,10 +204,51 @@ function Protocol (opts) {
 
 util.inherits(Protocol, duplexify)
 
+// extension api
+
+Protocol.prototype.remoteSupports = function (id) {
+  var self = this
+  if (!this._handshook) return
+
+  var offset = id - 64
```

Comment:
It might be good to pull this 64 out as a constant so that it doesn't look like a [magic number](http://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad).